src/views/home/index.tsx

```diff
@@ -27,7 +27,7 @@ function View(props: Props) {
       <Seo {...props} title={title} />
 
       {/* Visual Sections */}
-      <h1 className="absolute top-[-100px]">{title}</h1>
```

Comment:
Hey, this title shouldn't appear on the screen. So, it was put out the viewport since it can't be hidden due to a11y

Reply:
But this seems like a workaround, right? Why not let the text there, since when you actually create a home page, you'll have an h1 either way